ql/src/test/queries/clientpositive/materialized_view_create_rewrite_4.q

```diff
@@ -1,6 +1,6 @@
 -- Test Incremental rebuild of materialized view with aggregate but without count(*)
 -- when source tables have delete operations since last rebuild.
-
+--! qt:replace:/(totalSize\s+)(\S+|\s+|.+)/$1#Masked#/
```

Comment:
In this test, you put the replace after the explaining comments. At other tests, (e.g. at ql/src/test/queries/clientpositive/materialized_view_create_rewrite.q), before the comments. 
Could you please make it consistent?